Set ui.leds.enable to 0 by default for #459467.  No frequent wakeups on
laptops and desktop folks can enable this if they like it.  Patch based
on suggestion from أحمد المحمودي (Ahmed El-Mahmoudy).

Index: elinks/doc/html/elinks.conf.5.html
===================================================================
--- elinks.orig/doc/html/elinks.conf.5.html	2017-11-01 01:08:10.671908416 +0100
+++ elinks/doc/html/elinks.conf.5.html	2017-11-01 01:08:10.671908416 +0100
@@ -1456,7 +1456,7 @@
 Requires that underlining is enabled for the terminal.
 </dd></dl><h3 id="ui.leds">ui.leds (LEDs)</h3>
 <p>LEDs (visual indicators) options.
-</p><dl><dt id="ui.leds.enable">ui.leds.enable [0|1] (default: 1)</dt><dd>Enable LEDs. These visual indicators will inform you about various
+</p><dl><dt id="ui.leds.enable">ui.leds.enable [0|1] (default: 0)</dt><dd>Enable LEDs. These visual indicators will inform you about various
 states.
 </dd></dl><h3 id="ui.sessions">ui.sessions (Sessions)</h3>
 <p>Sessions settings.
Index: elinks/doc/man/man5/elinks.conf.5
===================================================================
--- elinks.orig/doc/man/man5/elinks.conf.5	2017-11-01 01:08:10.671908416 +0100
+++ elinks/doc/man/man5/elinks.conf.5	2017-11-01 01:08:10.671908416 +0100
@@ -2839,7 +2839,7 @@
 .SS "ui\&.leds (LEDs)"
 LEDs (visual indicators) options\&.
 .PP
-ui\&.leds\&.enable \fB[0|1]\fR (default: 1)
+ui\&.leds\&.enable \fB[0|1]\fR (default: 0)
 .RS 4
 Enable LEDs\&. These visual indicators will inform you about various states\&.
 .RE
Index: elinks/src/bfu/leds.c
===================================================================
--- elinks.orig/src/bfu/leds.c	2017-11-01 01:08:10.671908416 +0100
+++ elinks/src/bfu/leds.c	2017-11-01 01:08:10.671908416 +0100
@@ -100,7 +100,7 @@
 		N_("LEDs (visual indicators) options.")),
 
 	INIT_OPT_BOOL("ui.leds", N_("Enable"),
-		"enable", 0, 1,
+		"enable", 0, 0,
 		N_("Enable LEDs. These visual indicators will inform you "
 		"about various states.")),
 
